recipes/geant4/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set version = "10.5.0" %}
+
+package:
+    name: geant4
+    version: {{ version }}
+source:
+    url: http://cern.ch/geant4-data/releases/geant4.10.05.tar.gz
+    sha256: 2a86499d8327abc68456e5d7fc0303824e5704322291b331857cf4042286656e
+build:
+    number: '0'
+    skip: True  # [win]
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - make  # [unix]
+    host:
+        - expat
+        - xerces-c
+    run:
+        - expat
+        - xerces-c
+test:
+    requires:
+        - cmake
+        - make  # [unix]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+about:
+    summary: 'Geant4 is a toolkit for the simulation of the passage of particles through matter.'
+    dev_url: https://github.com/Geant4/geant4
```

As I already have this from the old GEANT4 PR:
```suggestion
    dev_url: https://github.com/Geant4/geant4
    description: |
        Geant4 is a toolkit for the simulation of the passage of particles through
        matter. Its areas of application include high energy, nuclear and
        accelerator physics, as well as studies in medical and space science. The
        three main reference papers for Geant4 are published in Nuclear Instruments
        and Methods in Physics Research [A 506 (2003) 250-303](http://www.sciencedirect.com/science/article/pii/S0168900203013688),
        IEEE Transactions on Nuclear Science [53 No. 1 (2006) 270-278](https://ieeexplore.ieee.org/xpls/abs_all.jsp?isnumber=33833&amp;arnumber=1610988&amp;count=33&amp;index=7)
        and Nuclear Instruments and Methods in Physics Research [A 835 (2016) 186-225](http://www.sciencedirect.com/science/article/pii/S0168900216306957).
```